README.md

```diff
@@ -8,3 +8,4 @@ If you are interested in collaborating with me, please pull-request with the for
 The following is a list of accepted Collaborators:
 
 * Fakhrurrozi, [mfakhru.github.io](https://mfakhru.github.io/).
+* Collaborator-1, [LinkedIn](https://www.linkedin/in/mfakhrurrozis).
```

Comment:
Hi, rucolab!
Your LinkedIn profile is interesting!

Reply:
thank you